SourceX/dx.cpp

```diff
@@ -61,9 +61,7 @@ void dx_create_primary_surface()
 #ifndef USE_SDL1
 	if (renderer) {
 		int width, height;
-		if (SDL_GetRendererOutputSize(renderer, &width, &height) <= -1) {
-			ErrSdl();
-		}
+		SDL_RenderGetLogicalSize(renderer, &width, &height);

```

Comment:
@AJenbo This is an important fix. Here is how the game looked without it:

![IMG_20191014_234813](https://user-images.githubusercontent.com/216339/66817884-5a125700-ef34-11e9-92fd-c44c49d6e0d5.jpg)

(output size here is 320x240, logical size 640x480)